AstSemantics.md

```diff
@@ -232,48 +232,58 @@ Since all AST nodes are expressions in WebAssembly, control constructs may yield
 a value and may appear as children of other expressions.
 
  * `nop`: an empty operator that does not yield a value 
- * `block`: a fixed-length sequence of expressions with a label at the end
+ * `block`: a sequence of expressions with a label at the end
  * `loop`: a block with an additional label at the beginning which may be used to form loops
+ * `multiblock`: a mixed sequence of expressions and `label` nodes, with a label at the end
+ * `label`: a node which defines a label and must be an immediate child of `multiblock`
  * `if`: if expression with a *then* expression
  * `if_else`: if expression with *then* and *else* expressions
- * `br`: branch to a given label in an enclosing construct
- * `br_if`: conditionally branch to a given label in an enclosing construct
- * `tableswitch`: a jump table which may jump either to an immediate `case`
-                  child or to a label in an enclosing construct
- * `case`: a case which must be an immediate child of `tableswitch`
+ * `br`: branch to a given label
```

Comment:
perhaps that comment wasn't clear enough as well ;) I mean, why not write "in a visible construct" using the new terminology?